src/Parser/Core/Modules/Items/ArchiveOfFaith.js

```diff
@@ -0,0 +1,57 @@
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+
+import Module from 'Parser/Core/Module';
+
+const debug = false;
+
+class ArchiveOfFaith extends Module {
+  archiveOfFaithCast = 0;
```

Comment:
nit: we're in the ArchiveOfFaith class, so being context aware this could be just `casts`